components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/css/idpmgt.css

```diff
@@ -57,3 +57,20 @@ color: #888;
 	padding-top: 5px;
 	display: block;
 }
+
+.templateName{
```

Comment:
It is better to introduce a separate CSS for the template, as this is going to be isolated and maybe ported to new UI as it is in near future.